modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -225,7 +225,7 @@
        (u.date/truncate unit)
        (u.date/add unit amount)
        (u.date/format-sql))
-   :timestamp])
+   :date])
```

Comment:
My understanding is that this function should never be called if we are interested in the time part, so it seems OK.
Still, I think we should make sure or at least emphasize (e.g., by changing the name from `datetime` to `date`) that no time information is kept.